Tools/PowershellModule/src/Library/New-WinGetSource.ps1

```diff
@@ -35,33 +35,33 @@ Function New-WinGetSource
     [Optional] The directory where Parameter objects will be created in. (Default: Current Directory)
 
     .PARAMETER RestSourcePath
-    [Optional] Path to the compiled Rest API Zip file. (Default: .\RestAPI\CompiledFunctions.ps1)
+    [Optional] Path to the compiled REST API Zip file. (Default: \Library\RestAPI\WinGet.RestSource.Functions.zip)
```

Comment:
Inaccurate? yourdefault is psscriptroot \ etc etc.

Reply:
That's just a user friendly version of it. The actual coded default is found in the lower section

Reply:
But its not user friendly if its inaccurate. Its a source of confusion

Reply:
```suggestion
    [Optional] Path to the compiled REST API Zip file. (Default: $PSScriptRoot\Library\RestAPI\WinGet.RestSource.Functions.zip)
```